.travis/before_install.sh

```diff
@@ -71,11 +71,18 @@ cd /tmp || die
         sudo apt-get install g++-multilib || die
     }
 
-    [ $BUILD_TARGET != posix-ncp ] || {
+    if [ $BUILD_TARGET == posix-ncp ] ||
+       [ $BUILD_TARGET == posix-distcheck ] ||
+       [ $BUILD_TARGET == posix-32-bit ] ||
+       [ $BUILD_TARGET == posix-ncp-spi ];
+    then
+        sudo -H pip install ipaddress || die
```

Comment:
Just avoid the conditional and put everything with the `pexpect` installation above.